### ISSUE TITLE
Why this and not the official release?

### ISSUE THREAD

**Issue by sntran** [sntran/GraphBLAS#1]
The official website of GraphBLAS provides a tarball extracted from SuiteSpare. There are also copies of SuiteSpare put up on GitHub by various people? So why does this repo exist?

There are a few reasons:
- In Elixir, I can't specify a tarball as dependency for the builder to use, and I don't want to manually keep GraphBLAS source codes in my project.
- The GitHub copies only contain official SuiteSpare releases. GraphBLAS is released on a more frequent schedule, and I want to use the latest.
- It's also easier to point the dependency directly to GraphBLAS instead of spare checking out a folder inside SuiteSpare.
- I want to see the diff between versions.
- The project I will be using GraphBLAS with will be public, so it needs to use public dependency.

In short, this is mostly for my personal use, but I leave this question here for reference.
